ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.scss

```diff
@@ -5,6 +5,40 @@
 
 @include angMatDialog.mainSet;
 
+::ng-deep.snackBarRestyle {
+  max-width: none;
+  background-color: $__success_msg_bg_transparent;
+  box-shadow: 0 2px 4px 0 $__success_msg_shadow;
+  padding: 0!important;
+  .successMessageContainer {
+    @include flexContainer;
+    height: 68px;
+    width: 352px;
+    color: $__Gunmetal;
+
+    aside {
+      @include flexContainer(center, center);
+      background-color: $__Medium_gea_green;
+      width: 48px;
+      height: 100%;
+    }
+    main {
+      @include flexContainer(normal, center, column);
+      width: 304px;
```

Comment:
**_#minor_*

Can be skipped as it resolved automatically by flex (you have common width and aside width).

Reply:
Done